docs/source/developer/build/BUILD_LINUX.md

```diff
@@ -1,9 +1,11 @@
 # Build Linux
 
-*Last Updated on April 11, 2020*
+*Last Updated on December 1, 2020*
 
 Please read the [general build guide](BUILD.md) for information on dependencies required for all platforms. Only Linux specific instructions are found in this file.
 
+You can use the [Vircadia Builder](https://github.com/kasenvr/vircadia-builder) to build on Linux more easily. Alternatively, you can follow the manual steps below.
```

Comment:
Vircadia-builder is still maintained on https://github.com/daleglass/vircadia-builder
Also this change links to kasenvr, but there is also a fork on vircadia. We can link to the vircadia fork and talk to dale about moving vircadia-builder to that eventually.
```suggestion
You can use the [Vircadia Builder](https://github.com/vircadia/vircadia-builder) to build on Linux more easily. Alternatively, you can follow the manual steps below.
```

Reply:
Yes, it would be good to make the vircadia-builder in the `vircadia` repo the master.
I'm reverting these changes in BUILD_LINUX.md because the file will be updated from the `vircadia` repo next user docs update.